test/e2e/mcc_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/openshift/machine-config-operator/test/framework"
+	"github.com/openshift/machine-config-operator/test/helpers"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Test case for https://github.com/openshift/machine-config-operator/issues/358
+func TestMCCPausedKubeletCAAlert(t *testing.T) {
+
+	var testPool = "master"
+
+	cs := framework.NewClientSet("")
+
+	// Get the machine config pool
+	mcp, err := cs.MachineConfigPools().Get(context.TODO(), testPool, metav1.GetOptions{})
+	require.Nil(t, err)
+
+	// Update our copy of the pool
+	newMcp := mcp.DeepCopy()
+	newMcp.Spec.Paused = true
+
+	t.Logf("Pausing pool")
+
+	// Update the pool to be paused
+	_, err = cs.MachineConfigPools().Update(context.TODO(), newMcp, metav1.UpdateOptions{})
+	require.Nil(t, err)
+	t.Logf("Paused")
+
+	t.Logf("Patching certificate")
+	// Take note that the slash had to be encoded as ~1 because it's a reference: https://www.rfc-editor.org/rfc/rfc6901#section-3
```

Comment:
This comment makes more sense in `helpers.ForceKubeApiserverCertificateRotation()` as opposed to here.

Reply:
Left that one behind when I broke that out into a function :(